test/wpool_process_SUITE.erl

```diff
@@ -175,24 +175,19 @@ continue(_Config) ->
 
 -spec format_status(config()) -> {comment, []}.
 format_status(_Config) ->
-    %% echo_server implements format_status/2
+    %% echo_server implements format_status/1
     {ok, Pid} = wpool_process:start_link(?MODULE, echo_server, {ok, state}, []),
-    %% therefore it returns {formatted_state, State} as its status
-    {status, Pid, {module, gen_server}, SItems} = sys:get_status(Pid),
-    [state] = [S || SItemList = [_ | _] <- SItems, {formatted_state, S} <- SItemList],
-    %% this code is actually what we use to retrieve the state in other tests
+    %% therefore it returns State as its status
     state = get_state(Pid),
     {comment, []}.
 
 -spec no_format_status(config()) -> {comment, []}.
 no_format_status(_Config) ->
-    %% crashy_server doesn't implement format_status/2
+    %% crashy_server doesn't implement format_status/1
     {ok, Pid} = wpool_process:start_link(?MODULE, crashy_server, state, []),
     %% therefore it uses the default format for the stauts (but with the status of
     %% the gen_server, not wpool_process)
-    {status, Pid, {module, gen_server}, SItems} = sys:get_status(Pid),
-    [state] =
-        [S || SItemList = [_ | _] <- SItems, {data, Data} <- SItemList, {"State", S} <- Data],
```

Comment:
Similar comment. Can't understand what's being tested...

Reply:
I guess the idea was to test that the result was the _unformatted_ status here… and the _formatted_ status in the other case.

Reply:
I'm not sure if that was the case, or if, after your changes, we're still testing that the worker can have format_status/1 defined or not… and that if it is not defined, we return whatever generic thing gen_server returns in that scenario.
So, I'll trust you… if you resolve this conversation, I will assume that everything is fine and tested :)